datatypes/src/collections/feature_collection.rs

```diff
@@ -805,6 +805,24 @@ where
 
         table_size + map_size
     }
+
+    /// Return the names of the columns of this type
+    pub fn column_names(&self, column_type: FeatureDataType) -> Vec<String> {
```

Comment:
I guess the method name should include something like filtered since we cannot get all column names here.

Reply:
or make the column_type optional?

Reply:
This is an option. However, it seems quite strange to get all column names by calling `collection.column_names(None)`.

Reply:
ec9ee747f1e86433c9466fe506e5ddf216651827